runtime/onert/core/include/compiler/train/LoweredTrainableGraph.h

```diff
@@ -50,6 +50,12 @@ class LoweredTrainableGraph : public ILoweredGraph
   compiler::GraphLowerInfo &lower_info() override { return _lower_info_map; }
   std::shared_ptr<ir::OperationIndexMap<int64_t>> indexed_ranks() { return _indexed_ranks; }
 
+  void setHasDynamicTensor(ir::OperationIndex, bool) override
+  {
+    throw std::runtime_error("LoweredTrainableGraph does not support dynamic tensors yet");
```

Comment:
```suggestion
  void setHasDynamicTensor(ir::OperationIndex, bool has_dynamic) override
  {
    if (has_dynamic)
        throw std::runtime_error("LoweredTrainableGraph does not support dynamic tensors yet");
    _has_dynamic_tensor_map.emplace(ind, val);
```

Reply:
It would be good not to change the existing behaviour for this function. So this change looks good to me. :)